src/main/java/nextstep/subway/fare/domain/policy/age/AgeFarePolicyType.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.subway.fare.domain.policy.age;
+
+import java.util.Arrays;
+import nextstep.subway.fare.domain.policy.age.impl.AdultAgeFarePolicy;
+import nextstep.subway.fare.domain.policy.age.impl.ChildAgeFarePolicy;
+import nextstep.subway.fare.domain.policy.age.impl.FreeAgeFarePolicy;
+import nextstep.subway.fare.domain.policy.age.impl.TeenagerAgeFarePolicy;
+
+public enum AgeFarePolicyType {
+
+    FREE(FreeAgeFarePolicy.class),
+    ADULT(AdultAgeFarePolicy.class),
+    CHILD(ChildAgeFarePolicy.class),
+    Teenager(TeenagerAgeFarePolicy.class);
```

Comment:
```suggestion
    TEENAGER(TeenagerAgeFarePolicy.class);
```

enum의 경우 어퍼케이스로 작성해주는 것이 관례입니다 :)

Reply:
😂